src/components/productsList/ProductsList.component.tsx

```diff
@@ -0,0 +1,16 @@
+import { ProductCard } from '@/components/productCard/ProductCard.component.tsx';
+import type { Product } from '@/interfaces/Product.ts';
+
+import styles from './productsList.module.css';
+
+export interface ProductsListComponentProps {
+    products: Product[];
+}
+
+export const ProductsListComponent: React.FC<ProductsListComponentProps> = ({ products }) => (
+    <div className={styles['products-list']}>
+        {products.map((product, index) => (
```

Comment:
As long as you don't have to remove / add new products you will not face any issues with using index as a key, but I will recommend to use `product.id` instead. 
```suggestion
        {products.map((product) => (
          <ProductCard key={product.id} product={product} />
```